internal/test/vote.go

```diff
@@ -0,0 +1,44 @@
+package test
+
+import (
+	"context"
+	"time"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+func MakeVote(
```

Comment:
There seems to be some overlapping of this function with the ones defined in:
* `types/test_util.go`
* `types/evidence_test.go`
* `evidence/verify_test.go`

It would be awesome if we took the opportunity to consolidate all three.

Reply:
Just as my comment above: up to you if you want to do it as part of this PR, or later